src/main/java/uk/gov/hmcts/reform/pip/subscription/management/service/SubscriptionLocationService.java

```diff
@@ -85,6 +94,29 @@ public String deleteSubscriptionByLocation(String locationId, String provenanceU
 
     }
 
+    private void deleteAllSubscriptionListTypeForLocation(List<Subscription> locationSubscriptions) {
+        List<String> uniqueUsers = locationSubscriptions.stream()
+            .map(Subscription::getUserId).distinct().toList();
+        List<UUID> subscriptionListTypesIds = new ArrayList<>();
+        int minNoOfSubscriptions = 1;
+
+        if (!uniqueUsers.isEmpty()) {
+
+            for (String userId : uniqueUsers) {
+                List<Subscription> userLocationSubscriptions = repository.findLocationSubscriptionsByUserId(userId);
+                if (userLocationSubscriptions.size() <= minNoOfSubscriptions) {
```

Comment:
I'm not sure the <= will work here, as if the user has 1 location subscription remaining, the list type config will still be deleted.

I believe this should be < 1, or == 0

Reply:
fixed.